develop/tutorials/code/com-liferay-docs-guestbook/gradle.properties

```diff
@@ -115,8 +115,8 @@
     #liferay.workspace.target.platform.version=7.0-GA7
 
     #
-    # Set the folder that contains Node.js-style theme projects. The default
-    # value is "themes".
+    # Set the folder that contains generated theme projects created with the 
```

Comment:
@codyhoag I just wanted to get your opinion on this. The `gradle.properties` file is auto-generated, correct? So we would need to change the template for this in the build tools to reflect the name change, right? Do you know how to do that?